### PR TITLE
chore: add logic for support single meta os taskpane create option

### PR DESCRIPTION
**Pull request by ZYUN-MSFT**
PM Spec [Here](https://microsoft.sharepoint.com/:w:/t/OfficePlatform/ESc9-gy6-ANOsKdGNfOm4hkBgh6fnjuQ9sezP7bpEHWB2w?e=JhfKqS)

### CODE REVIEW

**Comment by ZYUN-MSFT** [comment]
image

image

**Comment by hermanwenhe** [comment]
> > image
> > image
> 
> So Task Pane can be used across WXPO now correct? Can you share the next screen if I select Task Pane?

Next screen would be folder selection:

image

**Comment by ZYUN-MSFT** [comment]
image

image

This option will be avaliable for wxpo. And no programming lang need to be selected.

File: packages/fx-core/src/question/create.ts [hunk 1]
```diff
@@ -124,7 +124,9 @@ export function projectTypeQuestion(): SingleSelectQuestion {
           return [projectType];
         }
       } else if (getRuntime(inputs) === RuntimeOptions.NodeJS().id) {
-        if (featureFlagManager.getBooleanValue(FeatureFlags.OfficeAddin)) {
+        if (featureFlagManager.getBooleanValue(FeatureFlags.OfficeMetaOS)) {
+          staticOptions.push(ProjectTypeOptions.officeMetaOS(inputs.platform));
+        } else if (featureFlagManager.getBooleanValue(FeatureFlags.OfficeAddin)) {
```
Comment (tecton) [inline packages/fx-core/src/question/create.ts:129]: It looks like the FeatureFlags.OfficeMetaOS is replacing FeatureFlags.OfficeAddin, is it necessary to keep the office addin feature flag?

Comment (ZYUN-MSFT) [reply packages/fx-core/src/question/create.ts:129]: Not replace. The two feature gate control for two different create question options.

**Comment by hermanwenhe** [comment]
@tecton We will use a separate PR to clean these feature gates after WXP MOS public preview.

**Comment by MuyangAmigo** [comment]
Okay sounds good @ZYUN-MSFT @hermanwenhe Thanks!

**Review by jayzhang — APPROVED** [review]

**Pull request merged**